templates/moinmoin/files/setup/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Download MoinMoin
+curl -O http://static.moinmo.in/files/moin-1.9.10.tar.gz
+tar xvzf moin-1.9.10.tar.gz
+
+# Stage MoinMoin log files
+mv ~/setup/editlog.py moin-1.9.10/MoinMoin/logfile
+mv ~/setup/eventlog.py moin-1.9.10/MoinMoin/logfile
+
+# Move the app directory
+cp -a ~/moin-1.9.10/* /app/
+
+# Remove the install
+rm -rf ~/moin-1.9.10
+rm ~/moin-1.9.10.tar.gz
```

Comment:
I don't know if we're going to be updating this all that often, but you can curl the file to a specific name.  Then you don't need to include the version number throughout the file.  That's a better practice in general, I think.